WordPress/Classes/Models/WPAccount.m

```diff
@@ -116,14 +116,11 @@ - (NSArray *)visibleBlogs
     return sortedBlogs;
 }
 
-#pragma mark - API Helpers
+#pragma mark - WordPress.com support methods
 
-- (WordPressComApi *)restApi
+- (BOOL)isWPComAccount
 {
-    if (!_restApi) {
-        _restApi = [[WordPressComApi alloc] initWithOAuthToken:self.authToken];
-    }
-    return _restApi;
+    return self.restApi != nil;
```

Comment:
We may have a glitch here, since on 2nd launch the `_restApi` might not get initialized (since the authToken setter shouldn't be executed).